app/routes/_index.tsx

```diff
@@ -22,7 +21,7 @@ export const meta: V2_MetaFunction = () => {
 };
 
 export default function Index() {
-  const { sanityTekster } = useOutletContext<AppContext>();
+  const sanityTekster = useTekster();
   const tekster = sanityTekster[ESanitySteg.FORSIDE];
```

Comment:
Hva er fordelen med å gjøre det slikt? @sarahjelle

Reply:
Elis forklarte det bra. Må lese litt om kvirfor `return .... as ....` er bad practice.

Reply:
Tror ikke det er noen faktisk forskjell, men jeg syntes det så cleanere ut å bruke `useTekster()` for å bare hente tekstene fremfor å hente alt i context og plukke ut tekstene. Da brukes context likt for tekster og språk, og vi typene som var problemet når steg ble sendt inn er fortsatt løst ☺️